packages/pyright-internal/src/server.ts

```diff
@@ -52,6 +52,24 @@ class PyrightServer extends LanguageServerBase {
         this._controller = new CommandController(this);
     }
 
+    // Expands certain predefined variables supported within VS Code settings.
+    // Ideally, VS Code would provide an API for doing this expansion, but
+    // it doesn't. We'll handle the most common variables here as a convenience.
+    _expandPathVariables(rootPath: string, value: string): string {
```

Comment:
This should probably be `protected`.

Reply:
Python developer -- not use to having an actual formal `protected` :)

Reply:
No specifier would mean public, so it's not a huge problem for us, just a style nit. 🙂

Reply:
I've made this `protected expandPathVariables` now.